src/adhocracy_core/adhocracy_core/sheets/notification.py

```diff
@@ -37,6 +39,7 @@ class NotificationSchema(MappingSchema):
 
     follow_resources = UniqueReferences(reftype=NotificationFollowReference,
                                         choices_getter=get_follow_choices)
+    email_notification_enabled = Boolean(default=True, missing=drop)
```

Comment:
missing=drop means that if the value is None its not set at all. This allows setting just the required fields of a sheet.